charts/tekton-pipelines/templates/kaniko/trigger-template.yaml

```diff
@@ -23,6 +23,10 @@ spec:
       default: "."
       description: dockerfile context path
 
+    - name: kaniko_extra_args
```

Comment:
Maybe we can omit `kaniko` prefix and leave only `extra_args`? This variable is used only in `kaniko` pipelines and I think it is quite clear from `extra_args` and the rest context to what it belongs and what is it + we don't have any `kaniko` prefix for other vars in kaniko pipelines, I suggest to follow common approach

Reply:
`docker_file` or `docker_context` are obviously not relevant with `slack-notification` task for example. On the contrary `extra_args` is a super-generic name which at first glance can be equally relevant with any task: kustomize, argocd-deploy or slack-notification. When you just start analyzing relateions between tasks, pipelines, trigger bindings, etc. and look at `extra_args` you basically cannot have a good arguments to say "ok `extra_args` parameter is definitely cannot be an argument for `kustomize` task, I definietly cannot pay attention to it when I need to understand in details how `kustomize` task works".

Reply:
But we can, for example, rename both `kaniko` task parameter and pipeline parameter to `build_args` - it will be more unambiguous, because it is the only task in pipeline that performs builds.

Reply:
I like extra_args or kaniko_extra_args more than build_args, I see a lot of helm charts that are using term "extra". So I prefer the "extra" in naming